lib/features/mailbox_dashboard/presentation/controller/app_grid_dashboard_controller.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:core/presentation/state/failure.dart';
+import 'package:core/presentation/state/success.dart';
+import 'package:core/utils/app_logger.dart';
+import 'package:dartz/dartz.dart';
+import 'package:get/get_rx/get_rx.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/domain/app_dashboard/linagora_applications.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/domain/usecases/get_app_dashboard_configuration_interactor.dart';
+import 'package:tmail_ui_user/main/utils/app_config.dart';
+
+class AppGridDashboardController {
```

Comment:
why you are not extend from `GetxController`

Reply:
no need at the moment, so no implement :)